README.md

```diff
@@ -39,34 +39,45 @@ Firepad uses [Firebase](https://firebase.google.com) as a backend, so it require
 code. It can be added to any web app by including a few JavaScript files:
 
 ```HTML
-<!-- Firebase -->
-<script src="https://www.gstatic.com/firebasejs/3.0.3/firebase.js"></script>
-
-<!-- CodeMirror -->
-<script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.2.0/codemirror.js"></script>
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.2.0/codemirror.css"/>
-
-<!-- Firepad -->
-<link rel="stylesheet" href="https://cdn.firebase.com/libs/firepad/1.4.0/firepad.css" />
-<script src="https://cdn.firebase.com/libs/firepad/1.4.0/firepad.min.js"></script>
+<head>
+  <!-- Firebase -->
+  <script src="https://www.gstatic.com/firebasejs/3.3.0/firebase.js"></script>
+
+  <!-- CodeMirror -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.17.0/codemirror.js"></script>
```

Comment:
I upgraded this to the latest version and it all seems to work fine... All the demos worked for what I tested and in theory if they follow semver, we should be all good updating this.